requirements/requirements-llm.txt

```diff
@@ -1,3 +1,3 @@
 # optimum-amd[brevitas] @ git+https://github.com/huggingface/optimum-amd.git@main
 tqdm
-transformers[sentencepiece]==4.45.2
+transformers[sentencepiece]>=4.46.0
```

Comment:
Why have you bumped the version here? Does this no longer work with `transformers<4.46.0`?

Reply:
The test would be failing.